src/test/resources/features/search_contacts.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Search contacts
+
+  Background:
+    Given I log in Sales Force with valid User credentials
+    @createContact @deleteContact
+    Scenario: I search for the more recent created contact
+      When I navigate to Contacts page
+      And I search for the created contact
```

Comment:
I think this is for API because we can match web elements, but we can ask the trainer